test/mechanics/cell/CMakeLists.txt

```diff
@@ -14,3 +14,13 @@ add_unit_test(Matrix)
 
 add_unit_test(CellIpData
     mechanics/elements/ElementShapeFunctions.cpp)
+
+
+add_unit_test(PatchTest
+    mechanics/integrationtypes/IntegrationTypeBase.cpp
+    mechanics/integrationtypes/IntegrationTypeTensorProduct.cpp
+    math/Legendre.cpp # oh WHY? I do gauss... #SomeoneTODO
```

Comment:
The "standard" Gauss is more correctly called Gauss-Legendre quadrature, and your integration points are the roots of Legendre polynomials, implemented in `math/Legendre.cpp`. So you need this here. #NobodysTodo.